app/scripts/require_config.js

```diff
@@ -25,6 +25,7 @@ require.config({
     backbone: '../bower_components/backbone/backbone',
     canvasToBlob: '../bower_components/blueimp-canvas-to-blob/js/canvas-to-blob',
     chai: '../bower_components/chai/chai',
+    'fxa-checkbox': '../bower_components/fxa-checkbox/checkbox',
```

Comment:
Let's decide on one variant of camelCase, snake_case, and hyphenated. I advocate camelCase because that's the path of least resistance.